src/main/java/org/junit/rules/ExpectedException.java

```diff
@@ -14,7 +14,14 @@
 
 /**
  * The {@code ExpectedException} rule allows you to verify that your code
- * throws a specific exception.
+ * throws a specific exception. Note that
+ * {@link org.junit.Assert#assertThrows(java.lang.Class, org.junit.Assert.ThrowingRunnable)
+ * Assert.assertThrows}
+ * is often a better choice since it allows you to express exactly where you
```

Comment:
We should mention that this is only the case for Java 8. With Java < 8 you have to use anonymous classes that are hard to read.